lua/terrortown/lang/en.lua

```diff
@@ -1891,6 +1891,121 @@ L.label_voice_scaling_mode_linear = "Linear"
 L.label_voice_scaling_mode_power4 = "Power 4"
 L.label_voice_scaling_mode_log = "Logarithmic"
 
+-- 2023-10-19
+L.label_grenade_trajectory_ui = "Enable grenade trajectory indicator"
+L.label_grenade_fuse_meter_ui = "Enable grenade detonation fuse meter"
+
+L.label_weapon_tttbasegrenade_throw_force = "Throw Force"
+L.help_weapon_tttbasegrenade_throw_force = "How hard a grenade will be thrown, affects how quickly it will travel and how far it will go."
+L.label_weapon_tttbasegrenade_detonate_timer = "Detonate Time"
+L.help_weapon_tttbasegrenade_detonate_timer = "The time between the pin being pulled and the grenade exploding."
+L.label_weapon_tttbasegrenade_spherical_physics_radius = "Spherical Physics Radius"
+L.help_weapon_tttbasegrenade_spherical_physics_radius = [[The radius of the physics sphere.
+If 0, the projectile's physics will be based off its model rather than a sphere.]]
+L.label_weapon_tttbasegrenade_disarm_hit_players = "Disarm Hit Players"
+L.help_weapon_tttbasegrenade_disarm_hit_players = "If enabled, players hit by this grenade's projectile in-flight will drop their current weapon and unscope if possible."
+L.label_weapon_tttbasegrenade_damage_trigger_behavior = "Damage Trigger Behavior"
+L.help_weapon_tttbasegrenade_damage_trigger_behavior = [[Controls how loose throwables react to being damaged.
+ - None: Grenades will not react in any special way.
+ - Prime: Grenades will act like their pin is pulled, and will detonate after being damaged.
+ - Instant Explosion: Grenades will create their thrown version, and explode immediately.]]
```

Comment:
These two sound pretty equal. I think we need to rephrase these. Why do both detonate after being damaged? wouldn't the first one rather detonate after its timer has run out?

What does "create their thrown version" even mean? Also i think the average user does not know what that means, so we should simplify.

Reply:
no longer a concern in this fashion; but i still don't know how to distinguish between a swep grenade and an entity grenade.